acceptance/bundle/templates/lakeflow-pipelines/python/output/my_lakeflow_pipelines/resources/lakeflow_pipelines_etl.pipeline.yml

```diff
@@ -0,0 +1,21 @@
+# The main pipeline for my_lakeflow_pipelines
+
+resources:
+  pipelines:
+    lakeflow_pipelines_etl:
+      name: lakeflow_pipelines_etl
+      ## Catalog is required for serverless compute
+      catalog: ${var.catalog}
+      schema: ${var.schema}
+      serverless: true
+      root_path: "."
```

Comment:
Shouldn't rootpath be ".."? I assume it is relative to the `.pipeline.yml`.

Reply:
Ah, yes, this is old. It should say `../src/..._etl/`